grid/recruitment/create-candidate/maps/create-candidate.ts

```diff
@@ -152,6 +155,41 @@ export const createCandidateTest = (
           });
         }
       );
+
+      describeIf(
+        provider === 'workable',
+        'when specified subdomain does not exist',
+        () => {
+          let subdomain: string;
+
+          beforeAll(() => {
+            subdomain = process.env.WORKABLE_SUBDOMAIN!;
+
+            process.env.WORKABLE_SUBDOMAIN = 'invalid-superface';
+          });
+
+          afterAll(() => {
+            process.env.WORKABLE_SUBDOMAIN = subdomain;
+          });
+
+          it('returns error', async () => {
+            const result = await superface.run(
+              {
+                input: {
```

Comment:
question: Is there any reason why run method do not allow to pass parameters along with inputs of the perform?

Reply:
It would be much easier to change subdomain parameter in this test case.

Reply:
I agree, testing library could to be extended to do this. cc. @Jakub-Vacek

Reply:
So testing lib should allow passing parameters to `run` method? That sounds reasonable. I will looks into it.

Reply:
> So testing lib should allow passing parameters to run method? 
Exactly.

Reply:
Okey, I opened issue with feature request in testing repo. I will start working on it next week - it shouldn't be complicated.

Reply:
Thanks @Jakub-Vacek, I'm resolving this comment, we can get back to these tests after it'll be implemented.